apollo-core/src/main/java/com/ctrip/framework/apollo/core/ConfigConsts.java

```diff
@@ -19,11 +19,13 @@
 public interface ConfigConsts {
   String NAMESPACE_APPLICATION = "application";
   String CLUSTER_NAME_DEFAULT = "default";
-  String CLUSTER_NAMESPACE_SEPARATOR = "+";
+  String CLUSTER_NAMESPACE_SEPARATOR = "-";
```

Comment:
This change is very risky! It would invalidate the current cache since it uses `+` as the separator.
Please change it back!